subsys/bluetooth/mesh/provisioner.c

```diff
@@ -33,6 +33,7 @@
 #include "proxy.h"
 #include "prov.h"
 #include "settings.h"
+#include "keys.h"
```

Comment:
Could be hard understand which keys.h file this is including.

Reply:
Hm. Isn't this according to C language syntax? 
If compiler sees `#include "header.h"` then compiler looks for header file in the same folder where it is located at the current moment.
If compiler sees `#include <header.h>` then compiler looks for header file in the folders those have been provided as system or parameterized paths (via `-I`) .

Reply:
This is absolutely the same as it has been done for other Mesh headers. Please take a look at `access.h`, `cdb.h` etc etc